src/encoded/schemas/experiment_type.json

```diff
@@ -0,0 +1,151 @@
+            "default": "in review by lab",
+            "enum" : [
+                "released",
+                "in review by lab",
+                "deleted"
+            ]
+        },
+        "title": {
+            "title": "Experiment Type",
+            "description": "The display name for the experiment type",
+            "type": "string",
+            "lookup": 20
+        },
+        "experiment_name": {
+            "description": "DON'T SUBMIT, auto-generated",
+            "type": "string",
+            "pattern": "^[a-z0-9\\-]+$",
+            "uniqueKey": true,
+            "exclude_from": ["submit4dn", "FFedit-create"]
+        },
+        "controlled_term": {
+            "title": "Ontology Term",
+            "description": "Ontology term that matches this experiment type",
+            "type": "string",
+            "linkTo": "OntologyTerm"
+        },
+        "experiment_category": {
+            "title": "Experiment Category",
+            "description": "General category of the experiment type",
+            "type": "string",
+            "enum": ["Sequencing", "Microscopy"],
+            "lookup": 100
+        },
+        "assay_classification": {
+            "title": "Assay Classification",
+            "description": "Classification of assay type",
+            "type": "string",
+            "enum": [
+                "Linear DNA Enrichment",
+                "3C via Ligation",
+                "Ligation-free Multi-fragment 3C",
+                "RNA Quantification",
+                "RNA-DNA Proximity Capture",
+                "Fluorescence Localization"
+            ]
+        },
+        "assay_subclassification": {
+            "title": "Assay Sub-classification",
+            "description": "Sub-classification of assay based on what is measured",
+            "type": "string",
+            "enum": [
+                "Replication Timing",
+                "Proximity to Organelle",
+                "DNA Binding",
+                "Open Chromatin",
+                "DNA-DNA Pairwise Interactions",
+                "DNA-DNA Pairwise Interactions - Single Cell",
+                "DNA-DNA Multi-way Interactions",
+                "DNA-DNA Multi-way Interactions of Selected Loci",
+                "DNA-DNA Pairwise Interactions of Enriched Regions",
+                "DNA-DNA Pairwise Interactions of Selected Loci",
+                "RNA-DNA Multi-way Interactions",
+                "Fixed Sample DNA Localization",
+                "Fixed Sample RNA Localization",
+                "Time-Lapse"
+            ]
+        },
+        "other_tags": {
+            "title": "Other Tags",
+            "description": "Other tags that categorize or describe the experiment type",
+            "type": "array",
+            "items": {
+                "title": "Tag",
+                "description": "A tag for the item.",
+                "type":  "string"
+            }
+        },
+        "raw_files": {
```

Comment:
Maybe raw_file_types would be a better field name?
Again maybe an enum is too restrictive?   Would have to wait through a whole build cycle to add any microscopy raw file types.